### PR TITLE
Reader Search: Add train tracks

### PR DESCRIPTION
**Pull request by gibrown**
WIP. Currently only handles interactions and needs more testing.

Test live: https://calypso.live/?branch=add/reader-search-traintracks

### CODE REVIEW

File: client/reader/stats.js [hunk 1]
```diff
@@ -117,6 +117,7 @@ tracksRailcarEventWhitelist
 	.add( 'calypso_reader_article_commented_on' )
 	.add( 'calypso_reader_article_opened' )
 	.add( 'calypso_reader_startcard_clicked' )
+	.add( 'calypso_reader_search_clicked' )
```
Comment (blowery) [inline client/reader/stats.js]: maybe `calypso_reader_searchcard_clicked` just to differentiate from the search sidebar?

**Comment by blowery** [comment]
added in the traintracks renders, rebased to pick up changes to pick up site meta

**Comment by blowery** [comment]
added in the 'search' source for follows too

**Comment by blowery** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;s9BtJRG9&#x2F;9-send-train-tracks-events"> Send train tracks events</a>

**Comment by gibrown** [comment]
@blowery seems to test out ok, except for the follow button. The interact event is not firing when I follow something.

**Comment by blowery** [comment]
@gibrown should be sending them now

**Comment by gibrown** [comment]
LGTM and working now.

@bluefuton it seems likely that cold start needs to have the railcar added to Follow buttons also.

**Pull request merged**